code/modules/clothing/spacesuits/rig/rig_attackby.dm

```diff
@@ -201,7 +201,42 @@
 				else
 					to_chat(user, SPAN_WARNING("\The [S] is empty!"))
 
-		return
+	else if(istype(attacking_item,/obj/item/voidsuit_modkit/himeo)) // hardsuit kitting
```

Comment:
all of this should be done in `/obj/item/voidsuit_modkit`, and it needs to be generalized so that any voidsuit modkit can be made to work with hardsuits